src/app/models/admin_verification.server.model.ts

```diff
@@ -0,0 +1,98 @@
+import { Mongoose, Schema } from 'mongoose'
+
+import { IUserSchema } from 'src/types'
+
+import {
+  IAdminVerificationModel,
+  IAdminVerificationSchema,
+  UpsertOtpParams,
+} from '../../types/admin_verification'
```

Comment:
suggest exporting from `src/types/index` so we can import all types from the same place